stock_scheduler_assignation_horizon/views/res_config_settings_views.xml

```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+
+    <record id="res_config_settings_view_form_stock" model="ir.ui.view">
+        <field
+            name="name"
+        >res.config.settings.view.form.inherit.stock.logistics.workflow</field>
```

Comment:
```suggestion
```
This name seems copy pasted from another module.
In any case, defining the name on views is useless. Odoo generates it automatically